projects/js-packages/connection/components/connection-status-card/index.jsx

```diff
@@ -195,6 +200,8 @@ ConnectionStatusCard.propTypes = {
 	context: PropTypes.string,
 	/** Function to override default action for connect user account */
 	onConnectUser: PropTypes.func,
+	/** Shows an requires user connection message if true */
```

Comment:
Minor:

I'd call this prop `requiresUserConnection` instead of only `requiresConnection`

Also make it clear in the comment that will display the error if this prop is true AND user connection is missing

Reply:
Addressedini 2a0a1184e1d94b940de775215c2cbf3ce9a2b558